### DISCUSSION TITLE
Bending Lines: Maps and Data From Distortion to Deception (Leventhal Map Center 2020 exhibition)

### DISCUSSION THREAD

**Discussion by garrettdashnelson** [thegetty/quire#65] [Show & Tell]
Hi Quire community,

Back in early 2020, I had already started testing out Quire, having seen @geealbers present on it at the GRI's Streets of Los Angeles workshop. It turned out to be hugely serendipitous, because a few months later, I found myself confronting the task of moving a gallery exhibition planned at the [Leventhal Map & Education Center](https://leventhalmap.org) to an online format after the pandemic closures. Quire was absolutely crucial in making it possible for our very small team (primarily just myself and one other staff member) to pull together a rich and interactive exhibition in about two months.

You can see the exhibition here: https://www.leventhalmap.org/digital-exhibitions/bending-lines/

We did a fair amount of customization to the default Quire theme, reworking some of its nav components, and adding in custom features like an Education Tour and interactive captions for some materials, as well as fully-interactive exhibition pieces throughout the show. And while we didn't end up using Quire's linear-format PDF/EPUB capabilities in this first version, we're looking at creating a print catalog based on the Quire export at some point in the future.

I did all of the coding and design changes myself, so I'm happy to answer any questions from others pursuing similar projects!

Garrett Nelson
Curator of Maps & Director of Geographic Scholarship 
Leventhal Map & Education Center at the Boston Public Library

**Comment by geealbers** [comment]
Thanks for sharing this @garrettdashnelson! The quick pivot you had to make from physical to digital interaction is a great use case for Quire. One of my favorite things about this publication is the Education Tour. For anyone looking for ways of offering alternate paths through their Quire publications (which by design otherwise default to a very linear book-like structure) I highly recommend looking at what Garrett did. You can start at https://www.leventhalmap.org/digital-exhibitions/bending-lines/education-activities/ and like the graphic says "Follow me!". The links take you through six or seven objects in the exhibition with special education-focused texts. It's like a mini exhibition within the exhibition.

<img width="702" alt="Screen Shot 2021-03-18 at 10 40 12 AM" src="https://user-images.githubusercontent.com/7796401/111664186-c13dd600-87ce-11eb-985f-7feda92971b4.png">

Garrett, one question I had for you was on the use of the link boxes that say "Jump down to the education tour". Was there a reason you employed those rather than just linking with an anchor directly to the Education Tour callout box on each page? Just curious.
